readme.md

```diff
@@ -219,6 +220,24 @@ Integration tests are run under the `it` profile with the Failsafe plugin using
 ---
 
 ## Changelog
+### 4.0.0-beta.1 (Beta Release)
+**Disclaimer:** This is a beta release, and the feature has complete functionality. However, it is still undergoing testing to ensure optimal performance. Your feedback is valuable in helping us enhance the overall user experience.
+- JENKINS-66581 Add support for pull request builds
+    - Added pull requests tab to SCM categories
```

Comment:
I initially added it here as I thought it would be good to have an accessible detailed list of changes for anyone interested but happy to move it into the issue ticket instead if that's preferable. 😄